test/solidity-ibc/FixtureTest.t.sol

```diff
@@ -45,7 +47,31 @@ abstract contract FixtureTest is Test {
     }
 
     function setUp() public {
-        ics26Router = new ICS26Router(address(this));
+        // ============ Step 1: Deploy the logic contracts ==============
```

Comment:
It looks like we do this a lot, can we refactor it out somehow? Not a big deal if it is not easy, but would be nice to have a single place where we do instantiation.

Reply:
I think this is not easy at the moment. The ideal scenario would be to use the deploy script but I'm not sure if it could be done. Happy to review any PR that does this